<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-145318313-2"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-145318313-2');
</script>


<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<script>
     (adsbygoogle = window.adsbygoogle || []).push({
          google_ad_client: "ca-pub-3848514720264642",
          enable_page_level_ads: true
     });
</script>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 6 Bioconductor | BioSakshat - Free Study Materials</title>
  <meta name="description" content="+++" />
  <meta name="generator" content="bookdown 0.12 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 6 Bioconductor | BioSakshat - Free Study Materials" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="+++" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 6 Bioconductor | BioSakshat - Free Study Materials" />
  
  <meta name="twitter:description" content="+++" />
  

<meta name="author" content="By Priyabrata Panigrahi and Pandurang Kolekar" />


<meta name="date" content="2019-08-13" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="data-visualization.html">
<link rel="next" href="r-case-study-and-tasks.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-1.3/htmlwidgets.js"></script>
<link href="libs/vis-4.20.1/vis.css" rel="stylesheet" />
<script src="libs/vis-4.20.1/vis.min.js"></script>
<script src="libs/visNetwork-binding-2.0.7/visNetwork.js"></script>


<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">BioSakshat</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Site navigation</a></li>
<li class="chapter" data-level="2" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html"><i class="fa fa-check"></i><b>2</b> Data types and Data structures</a><ul>
<li class="chapter" data-level="2.1" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#lets-begin"><i class="fa fa-check"></i><b>2.1</b> Lets Begin</a></li>
<li class="chapter" data-level="2.2" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#expression"><i class="fa fa-check"></i><b>2.2</b> Expression</a></li>
<li class="chapter" data-level="2.3" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#assignment"><i class="fa fa-check"></i><b>2.3</b> Assignment</a></li>
<li class="chapter" data-level="2.4" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#working-directory"><i class="fa fa-check"></i><b>2.4</b> Working directory</a></li>
<li class="chapter" data-level="2.5" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#list-of-objects-in-workspaceenvironment"><i class="fa fa-check"></i><b>2.5</b> List of objects in workspace/environment</a></li>
<li class="chapter" data-level="2.6" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#getting-help-in-r"><i class="fa fa-check"></i><b>2.6</b> getting help in R</a><ul>
<li class="chapter" data-level="2.6.1" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#some-things-to-remember"><i class="fa fa-check"></i><b>2.6.1</b> Some things to Remember</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#data-types"><i class="fa fa-check"></i><b>2.7</b> Data Types</a></li>
<li class="chapter" data-level="2.8" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#data-structures"><i class="fa fa-check"></i><b>2.8</b> Data Structures</a></li>
<li class="chapter" data-level="2.9" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#vector"><i class="fa fa-check"></i><b>2.9</b> Vector</a><ul>
<li class="chapter" data-level="2.9.1" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#creating-a-vector"><i class="fa fa-check"></i><b>2.9.1</b> Creating a vector</a></li>
<li class="chapter" data-level="2.9.2" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#create-vector-of-one-element"><i class="fa fa-check"></i><b>2.9.2</b> Create Vector of one element</a></li>
<li class="chapter" data-level="2.9.3" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#create-vector-using-range-operator"><i class="fa fa-check"></i><b>2.9.3</b> Create Vector using Range operator (:)</a></li>
<li class="chapter" data-level="2.9.4" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#create-vector-using-c"><i class="fa fa-check"></i><b>2.9.4</b> Create Vector using c()</a></li>
<li class="chapter" data-level="2.9.5" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#seq-function"><i class="fa fa-check"></i><b>2.9.5</b> seq() function</a></li>
<li class="chapter" data-level="2.9.6" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#rep-function"><i class="fa fa-check"></i><b>2.9.6</b> rep() function</a></li>
<li class="chapter" data-level="2.9.7" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#sample-function"><i class="fa fa-check"></i><b>2.9.7</b> sample() function</a></li>
<li class="chapter" data-level="2.9.8" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#rnorm-function"><i class="fa fa-check"></i><b>2.9.8</b> rnorm() function</a></li>
<li class="chapter" data-level="2.9.9" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#fetching-elements-from-a-vector"><i class="fa fa-check"></i><b>2.9.9</b> Fetching elements from a vector</a></li>
<li class="chapter" data-level="2.9.10" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#delete-elements-from-a-vector"><i class="fa fa-check"></i><b>2.9.10</b> Delete element(s) from a vector</a></li>
<li class="chapter" data-level="2.9.11" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#add-elements-to-existing-vector"><i class="fa fa-check"></i><b>2.9.11</b> Add element(s) to existing vector</a></li>
<li class="chapter" data-level="2.9.12" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#replace-elements-in-existing-vector"><i class="fa fa-check"></i><b>2.9.12</b> Replace elements in existing vector</a></li>
<li class="chapter" data-level="2.9.13" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#inbuilt-functions-for-numeric-vector"><i class="fa fa-check"></i><b>2.9.13</b> Inbuilt functions for numeric vector</a></li>
<li class="chapter" data-level="2.9.14" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#correlation-using-cor"><i class="fa fa-check"></i><b>2.9.14</b> Correlation using cor()</a></li>
<li class="chapter" data-level="2.9.15" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#set-operations"><i class="fa fa-check"></i><b>2.9.15</b> Set operations</a></li>
<li class="chapter" data-level="2.9.16" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#arithmetic-expressions"><i class="fa fa-check"></i><b>2.9.16</b> Arithmetic expressions</a></li>
<li class="chapter" data-level="2.9.17" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#arithmetic-operations-on-vector"><i class="fa fa-check"></i><b>2.9.17</b> Arithmetic operations on vector</a></li>
<li class="chapter" data-level="2.9.18" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#operator-precedence"><i class="fa fa-check"></i><b>2.9.18</b> Operator precedence</a></li>
<li class="chapter" data-level="2.9.19" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#conditional-statements"><i class="fa fa-check"></i><b>2.9.19</b> Conditional statements</a></li>
<li class="chapter" data-level="2.9.20" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#check-for-missing-values"><i class="fa fa-check"></i><b>2.9.20</b> Check for missing values</a></li>
<li class="chapter" data-level="2.9.21" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#check-data-types-and-data-structures"><i class="fa fa-check"></i><b>2.9.21</b> Check data types and data structures</a></li>
<li class="chapter" data-level="2.9.22" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#implicit-data-type-conversion"><i class="fa fa-check"></i><b>2.9.22</b> Implicit Data type conversion</a></li>
<li class="chapter" data-level="2.9.23" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#character-vector-and-related-functions"><i class="fa fa-check"></i><b>2.9.23</b> Character vector and related functions</a></li>
<li class="chapter" data-level="2.9.24" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#explicit-data-type-conversion"><i class="fa fa-check"></i><b>2.9.24</b> Explicit Data Type Conversion</a></li>
</ul></li>
<li class="chapter" data-level="2.10" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#matrix"><i class="fa fa-check"></i><b>2.10</b> Matrix</a><ul>
<li class="chapter" data-level="2.10.1" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#create-a-matrix-from-vector-using-dim-attributes"><i class="fa fa-check"></i><b>2.10.1</b> Create a matrix from vector using dim attributes</a></li>
<li class="chapter" data-level="2.10.2" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#create-matrix-using-matrix"><i class="fa fa-check"></i><b>2.10.2</b> Create matrix using matrix()</a></li>
<li class="chapter" data-level="2.10.3" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#functions-that-can-be-applied-to-a-matrix"><i class="fa fa-check"></i><b>2.10.3</b> Functions that can be applied to a matrix</a></li>
<li class="chapter" data-level="2.10.4" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#assign-columnrow-names"><i class="fa fa-check"></i><b>2.10.4</b> Assign column/row names</a></li>
<li class="chapter" data-level="2.10.5" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#fetch-elements-from-a-matrix-using-indices"><i class="fa fa-check"></i><b>2.10.5</b> Fetch elements from a matrix using indices</a></li>
<li class="chapter" data-level="2.10.6" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#fetch-element-using-index-matrix"><i class="fa fa-check"></i><b>2.10.6</b> Fetch element using index matrix</a></li>
<li class="chapter" data-level="2.10.7" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#fetch-elements-from-a-matrix-using-row-and-column-names"><i class="fa fa-check"></i><b>2.10.7</b> Fetch elements from a matrix using row and column names</a></li>
<li class="chapter" data-level="2.10.8" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#insert-rows-and-columns-by-rbind-and-cbind"><i class="fa fa-check"></i><b>2.10.8</b> Insert rows and columns by rbind() and cbind()</a></li>
<li class="chapter" data-level="2.10.9" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#create-matrix-using-rbind-and-cbind"><i class="fa fa-check"></i><b>2.10.9</b> Create matrix using rbind() and cbind()</a></li>
<li class="chapter" data-level="2.10.10" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#delete-rows-and-columns"><i class="fa fa-check"></i><b>2.10.10</b> Delete row(s) and column(s)</a></li>
<li class="chapter" data-level="2.10.11" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#check-for-matrix"><i class="fa fa-check"></i><b>2.10.11</b> Check for matrix</a></li>
<li class="chapter" data-level="2.10.12" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#conditional-statements-on-matrix"><i class="fa fa-check"></i><b>2.10.12</b> Conditional statements on matrix</a></li>
</ul></li>
<li class="chapter" data-level="2.11" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#data-frame"><i class="fa fa-check"></i><b>2.11</b> Data Frame</a><ul>
<li class="chapter" data-level="2.11.1" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#create-a-data-frame"><i class="fa fa-check"></i><b>2.11.1</b> Create a data frame</a></li>
<li class="chapter" data-level="2.11.2" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#fetch-values-from-a-data-frame"><i class="fa fa-check"></i><b>2.11.2</b> Fetch values from a data frame</a></li>
<li class="chapter" data-level="2.11.3" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#insert-column-to-data-frame"><i class="fa fa-check"></i><b>2.11.3</b> Insert column to data frame</a></li>
<li class="chapter" data-level="2.11.4" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#delete-column-or-row-from-data-frame"><i class="fa fa-check"></i><b>2.11.4</b> Delete column or row from data frame</a></li>
<li class="chapter" data-level="2.11.5" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#check-for-data-frame"><i class="fa fa-check"></i><b>2.11.5</b> Check for data frame</a></li>
</ul></li>
<li class="chapter" data-level="2.12" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#list"><i class="fa fa-check"></i><b>2.12</b> List</a><ul>
<li class="chapter" data-level="2.12.1" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#access-component-of-list-using-notation"><i class="fa fa-check"></i><b>2.12.1</b> Access component of list using $ notation</a></li>
<li class="chapter" data-level="2.12.2" data-path="data-types-and-data-structures.html"><a href="data-types-and-data-structures.html#access-component-of-list-using"><i class="fa fa-check"></i><b>2.12.2</b> Access component of list using []</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html"><i class="fa fa-check"></i><b>3</b> If/For/Function/IO</a><ul>
<li class="chapter" data-level="3.1" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#conditional-statements-1"><i class="fa fa-check"></i><b>3.1</b> Conditional statements</a></li>
<li class="chapter" data-level="3.2" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#repetitive-execution-using-for-loop"><i class="fa fa-check"></i><b>3.2</b> Repetitive execution using for loop</a><ul>
<li class="chapter" data-level="3.2.1" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#task"><i class="fa fa-check"></i><b>3.2.1</b> Task</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#functions"><i class="fa fa-check"></i><b>3.3</b> Functions</a><ul>
<li class="chapter" data-level="3.3.1" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#define-function"><i class="fa fa-check"></i><b>3.3.1</b> Define function</a></li>
<li class="chapter" data-level="3.3.2" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#call-function"><i class="fa fa-check"></i><b>3.3.2</b> Call function</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#file-io"><i class="fa fa-check"></i><b>3.4</b> File IO</a><ul>
<li class="chapter" data-level="3.4.1" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#the-read.table-function"><i class="fa fa-check"></i><b>3.4.1</b> The read.table() function</a></li>
<li class="chapter" data-level="3.4.2" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#read-data"><i class="fa fa-check"></i><b>3.4.2</b> Read data</a></li>
<li class="chapter" data-level="3.4.3" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#about-headertrue-parameter"><i class="fa fa-check"></i><b>3.4.3</b> About header=TRUE parameter</a></li>
<li class="chapter" data-level="3.4.4" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#about-stringsasfactors-false-parameter"><i class="fa fa-check"></i><b>3.4.4</b> About stringsAsFactors = FALSE parameter</a></li>
<li class="chapter" data-level="3.4.5" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#other-useful-parameters"><i class="fa fa-check"></i><b>3.4.5</b> Other useful parameters:</a></li>
<li class="chapter" data-level="3.4.6" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#read-file-with-unequal-columns-in-first-row-using-read.table."><i class="fa fa-check"></i><b>3.4.6</b> Read file with unequal columns in first row using read.table.</a></li>
<li class="chapter" data-level="3.4.7" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#read-file-consisting-of-comments-blank-lines-null-values"><i class="fa fa-check"></i><b>3.4.7</b> Read file consisting of comments, blank lines, null values</a></li>
<li class="chapter" data-level="3.4.8" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#read-csv-file-read.csv"><i class="fa fa-check"></i><b>3.4.8</b> Read csv file read.csv()</a></li>
<li class="chapter" data-level="3.4.9" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#reading-data-from-excel-file"><i class="fa fa-check"></i><b>3.4.9</b> Reading data from excel file</a></li>
<li class="chapter" data-level="3.4.10" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#read-data-using-readlines"><i class="fa fa-check"></i><b>3.4.10</b> Read data using readLines()</a></li>
<li class="chapter" data-level="3.4.11" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#read-data-using-clipboard-feature"><i class="fa fa-check"></i><b>3.4.11</b> Read data using clipboard feature</a></li>
<li class="chapter" data-level="3.4.12" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#view-data-frame-using-view"><i class="fa fa-check"></i><b>3.4.12</b> View data frame using View()</a></li>
<li class="chapter" data-level="3.4.13" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#edit-data-frame-using-edit"><i class="fa fa-check"></i><b>3.4.13</b> Edit data frame using edit()</a></li>
<li class="chapter" data-level="3.4.14" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#write-r-data-frames-in-a-file-using-write.table"><i class="fa fa-check"></i><b>3.4.14</b> Write R data frames in a file using write.table()</a></li>
<li class="chapter" data-level="3.4.15" data-path="ifforfunctionio.html"><a href="ifforfunctionio.html#write-using-cat"><i class="fa fa-check"></i><b>3.4.15</b> Write using cat()</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="data-manipulation.html"><a href="data-manipulation.html"><i class="fa fa-check"></i><b>4</b> Data Manipulation</a><ul>
<li class="chapter" data-level="4.1" data-path="data-manipulation.html"><a href="data-manipulation.html#the-apply-function"><i class="fa fa-check"></i><b>4.1</b> The apply() function</a></li>
<li class="chapter" data-level="4.2" data-path="data-manipulation.html"><a href="data-manipulation.html#sapply-function"><i class="fa fa-check"></i><b>4.2</b> sapply() function</a></li>
<li class="chapter" data-level="4.3" data-path="data-manipulation.html"><a href="data-manipulation.html#lapply-function"><i class="fa fa-check"></i><b>4.3</b> lapply() function</a></li>
<li class="chapter" data-level="4.4" data-path="data-manipulation.html"><a href="data-manipulation.html#dplyr-package"><i class="fa fa-check"></i><b>4.4</b> dplyr package</a><ul>
<li class="chapter" data-level="4.4.1" data-path="data-manipulation.html"><a href="data-manipulation.html#load-dplyr-library"><i class="fa fa-check"></i><b>4.4.1</b> Load dplyr library</a></li>
<li class="chapter" data-level="4.4.2" data-path="data-manipulation.html"><a href="data-manipulation.html#load-gene-expression-data"><i class="fa fa-check"></i><b>4.4.2</b> Load Gene Expression Data</a></li>
<li class="chapter" data-level="4.4.3" data-path="data-manipulation.html"><a href="data-manipulation.html#filter-rows-with-filter"><i class="fa fa-check"></i><b>4.4.3</b> Filter rows with filter()</a></li>
<li class="chapter" data-level="4.4.4" data-path="data-manipulation.html"><a href="data-manipulation.html#piping-using"><i class="fa fa-check"></i><b>4.4.4</b> Piping using %&gt;%</a></li>
<li class="chapter" data-level="4.4.5" data-path="data-manipulation.html"><a href="data-manipulation.html#arrange-rows-with-arrange"><i class="fa fa-check"></i><b>4.4.5</b> Arrange rows with arrange()</a></li>
<li class="chapter" data-level="4.4.6" data-path="data-manipulation.html"><a href="data-manipulation.html#select-rows-position-wise-using-slice"><i class="fa fa-check"></i><b>4.4.6</b> Select rows position wise using slice()</a></li>
<li class="chapter" data-level="4.4.7" data-path="data-manipulation.html"><a href="data-manipulation.html#select-column-using-select"><i class="fa fa-check"></i><b>4.4.7</b> Select column using select()</a></li>
<li class="chapter" data-level="4.4.8" data-path="data-manipulation.html"><a href="data-manipulation.html#extract-unique-rows-using-distinct"><i class="fa fa-check"></i><b>4.4.8</b> Extract unique rows using distinct()</a></li>
<li class="chapter" data-level="4.4.9" data-path="data-manipulation.html"><a href="data-manipulation.html#add-new-columns-with-mutate-and-transmute-function"><i class="fa fa-check"></i><b>4.4.9</b> Add new columns with mutate() and transmute() function</a></li>
<li class="chapter" data-level="4.4.10" data-path="data-manipulation.html"><a href="data-manipulation.html#summarise-values-with-summarise"><i class="fa fa-check"></i><b>4.4.10</b> Summarise values with summarise()</a></li>
<li class="chapter" data-level="4.4.11" data-path="data-manipulation.html"><a href="data-manipulation.html#grouped-operations"><i class="fa fa-check"></i><b>4.4.11</b> Grouped operations</a></li>
<li class="chapter" data-level="4.4.12" data-path="data-manipulation.html"><a href="data-manipulation.html#joins-two-table"><i class="fa fa-check"></i><b>4.4.12</b> Joins two table</a></li>
<li class="chapter" data-level="4.4.13" data-path="data-manipulation.html"><a href="data-manipulation.html#references"><i class="fa fa-check"></i><b>4.4.13</b> References</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="data-visualization.html"><a href="data-visualization.html"><i class="fa fa-check"></i><b>5</b> Data Visualization</a><ul>
<li class="chapter" data-level="5.1" data-path="data-visualization.html"><a href="data-visualization.html#base-graphics"><i class="fa fa-check"></i><b>5.1</b> Base Graphics</a><ul>
<li class="chapter" data-level="5.1.1" data-path="data-visualization.html"><a href="data-visualization.html#simple-scatter-plot"><i class="fa fa-check"></i><b>5.1.1</b> Simple scatter plot</a></li>
<li class="chapter" data-level="5.1.2" data-path="data-visualization.html"><a href="data-visualization.html#saving-plot"><i class="fa fa-check"></i><b>5.1.2</b> Saving plot</a></li>
<li class="chapter" data-level="5.1.3" data-path="data-visualization.html"><a href="data-visualization.html#add-titlesubtitlex-axis-labely-axis-label-to-plot"><i class="fa fa-check"></i><b>5.1.3</b> Add title/subtitle/x-axis label/y-axis label to plot</a></li>
<li class="chapter" data-level="5.1.4" data-path="data-visualization.html"><a href="data-visualization.html#increase-line-thickness"><i class="fa fa-check"></i><b>5.1.4</b> Increase line thickness</a></li>
<li class="chapter" data-level="5.1.5" data-path="data-visualization.html"><a href="data-visualization.html#assign-color-to-lines-and-points"><i class="fa fa-check"></i><b>5.1.5</b> Assign color to lines and points</a></li>
<li class="chapter" data-level="5.1.6" data-path="data-visualization.html"><a href="data-visualization.html#explore-built-in-colors"><i class="fa fa-check"></i><b>5.1.6</b> Explore built-in colors</a></li>
<li class="chapter" data-level="5.1.7" data-path="data-visualization.html"><a href="data-visualization.html#explore-different-symbols"><i class="fa fa-check"></i><b>5.1.7</b> Explore different symbols</a></li>
<li class="chapter" data-level="5.1.8" data-path="data-visualization.html"><a href="data-visualization.html#explore-different-line-types"><i class="fa fa-check"></i><b>5.1.8</b> Explore different line types</a></li>
<li class="chapter" data-level="5.1.9" data-path="data-visualization.html"><a href="data-visualization.html#rescale-the-x-axis-and-y-axis"><i class="fa fa-check"></i><b>5.1.9</b> Rescale the x-axis and y-axis</a></li>
<li class="chapter" data-level="5.1.10" data-path="data-visualization.html"><a href="data-visualization.html#magnification-of-labels-and-symbols"><i class="fa fa-check"></i><b>5.1.10</b> Magnification of labels and symbols</a></li>
<li class="chapter" data-level="5.1.11" data-path="data-visualization.html"><a href="data-visualization.html#explore-bty-box-type"><i class="fa fa-check"></i><b>5.1.11</b> Explore bty (box type)</a></li>
<li class="chapter" data-level="5.1.12" data-path="data-visualization.html"><a href="data-visualization.html#explore-las-style-of-axis-labels"><i class="fa fa-check"></i><b>5.1.12</b> Explore las (style of axis labels)</a></li>
<li class="chapter" data-level="5.1.13" data-path="data-visualization.html"><a href="data-visualization.html#explore-col.axis-col.lab-col.main-col.sub-parameters"><i class="fa fa-check"></i><b>5.1.13</b> Explore col.axis, col.lab, col.main, col.sub parameters</a></li>
<li class="chapter" data-level="5.1.14" data-path="data-visualization.html"><a href="data-visualization.html#low-level-plotting-functions-lines"><i class="fa fa-check"></i><b>5.1.14</b> Low level plotting functions (lines)</a></li>
<li class="chapter" data-level="5.1.15" data-path="data-visualization.html"><a href="data-visualization.html#low-level-plotting-functions-legend"><i class="fa fa-check"></i><b>5.1.15</b> Low level plotting functions (legend)</a></li>
<li class="chapter" data-level="5.1.16" data-path="data-visualization.html"><a href="data-visualization.html#adding-texts-to-existing-plot-text"><i class="fa fa-check"></i><b>5.1.16</b> Adding texts to existing plot (text)</a></li>
<li class="chapter" data-level="5.1.17" data-path="data-visualization.html"><a href="data-visualization.html#explore-line-plot"><i class="fa fa-check"></i><b>5.1.17</b> Explore Line Plot</a></li>
<li class="chapter" data-level="5.1.18" data-path="data-visualization.html"><a href="data-visualization.html#explore-simple-bar-plot"><i class="fa fa-check"></i><b>5.1.18</b> Explore simple Bar plot</a></li>
<li class="chapter" data-level="5.1.19" data-path="data-visualization.html"><a href="data-visualization.html#explore-stacked-bar-plot"><i class="fa fa-check"></i><b>5.1.19</b> Explore stacked bar plot</a></li>
<li class="chapter" data-level="5.1.20" data-path="data-visualization.html"><a href="data-visualization.html#grouped-bar-plots"><i class="fa fa-check"></i><b>5.1.20</b> Grouped Bar plots</a></li>
<li class="chapter" data-level="5.1.21" data-path="data-visualization.html"><a href="data-visualization.html#histogram"><i class="fa fa-check"></i><b>5.1.21</b> Histogram</a></li>
<li class="chapter" data-level="5.1.22" data-path="data-visualization.html"><a href="data-visualization.html#density-plot"><i class="fa fa-check"></i><b>5.1.22</b> Density plot</a></li>
<li class="chapter" data-level="5.1.23" data-path="data-visualization.html"><a href="data-visualization.html#plot-density-plot-of-two-distributions"><i class="fa fa-check"></i><b>5.1.23</b> Plot density plot of two distributions</a></li>
<li class="chapter" data-level="5.1.24" data-path="data-visualization.html"><a href="data-visualization.html#box-plots"><i class="fa fa-check"></i><b>5.1.24</b> Box plots</a></li>
<li class="chapter" data-level="5.1.25" data-path="data-visualization.html"><a href="data-visualization.html#pie-chart"><i class="fa fa-check"></i><b>5.1.25</b> Pie chart</a></li>
<li class="chapter" data-level="5.1.26" data-path="data-visualization.html"><a href="data-visualization.html#venn-diagram"><i class="fa fa-check"></i><b>5.1.26</b> Venn diagram</a></li>
<li class="chapter" data-level="5.1.27" data-path="data-visualization.html"><a href="data-visualization.html#heatmap"><i class="fa fa-check"></i><b>5.1.27</b> Heatmap</a></li>
<li class="chapter" data-level="5.1.28" data-path="data-visualization.html"><a href="data-visualization.html#scatter-plot-matrices"><i class="fa fa-check"></i><b>5.1.28</b> Scatter plot Matrices</a></li>
<li class="chapter" data-level="5.1.29" data-path="data-visualization.html"><a href="data-visualization.html#ballon-plot"><i class="fa fa-check"></i><b>5.1.29</b> Ballon plot</a></li>
<li class="chapter" data-level="5.1.30" data-path="data-visualization.html"><a href="data-visualization.html#multi-panel-plots"><i class="fa fa-check"></i><b>5.1.30</b> Multi panel plots</a></li>
<li class="chapter" data-level="5.1.31" data-path="data-visualization.html"><a href="data-visualization.html#pca-prinicipal-component-analysis"><i class="fa fa-check"></i><b>5.1.31</b> PCA (Prinicipal Component Analysis)</a></li>
<li class="chapter" data-level="5.1.32" data-path="data-visualization.html"><a href="data-visualization.html#classical-metric-multidimensional-scaling"><i class="fa fa-check"></i><b>5.1.32</b> Classical (Metric) Multidimensional Scaling</a></li>
<li class="chapter" data-level="5.1.33" data-path="data-visualization.html"><a href="data-visualization.html#plotting-k-means"><i class="fa fa-check"></i><b>5.1.33</b> Plotting K-means</a></li>
<li class="chapter" data-level="5.1.34" data-path="data-visualization.html"><a href="data-visualization.html#dendrogram"><i class="fa fa-check"></i><b>5.1.34</b> Dendrogram</a></li>
<li class="chapter" data-level="5.1.35" data-path="data-visualization.html"><a href="data-visualization.html#network-graphs"><i class="fa fa-check"></i><b>5.1.35</b> Network graphs</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="data-visualization.html"><a href="data-visualization.html#plot-using-ggplot2"><i class="fa fa-check"></i><b>5.2</b> Plot using ggplot2</a><ul>
<li class="chapter" data-level="5.2.1" data-path="data-visualization.html"><a href="data-visualization.html#scatter-plot"><i class="fa fa-check"></i><b>5.2.1</b> Scatter plot</a></li>
<li class="chapter" data-level="5.2.2" data-path="data-visualization.html"><a href="data-visualization.html#add-geom_point-layer"><i class="fa fa-check"></i><b>5.2.2</b> Add geom_point() layer</a></li>
<li class="chapter" data-level="5.2.3" data-path="data-visualization.html"><a href="data-visualization.html#add-geom_smooth-layer-linear-modeling"><i class="fa fa-check"></i><b>5.2.3</b> Add geom_smooth() layer, linear modeling</a></li>
<li class="chapter" data-level="5.2.4" data-path="data-visualization.html"><a href="data-visualization.html#explore-aesthetic-parameter-col"><i class="fa fa-check"></i><b>5.2.4</b> Explore aesthetic parameter “col”</a></li>
<li class="chapter" data-level="5.2.5" data-path="data-visualization.html"><a href="data-visualization.html#assign-aes-to-individual-layer"><i class="fa fa-check"></i><b>5.2.5</b> Assign aes() to individual layer</a></li>
<li class="chapter" data-level="5.2.6" data-path="data-visualization.html"><a href="data-visualization.html#explore-aesthetic-parameter-shape"><i class="fa fa-check"></i><b>5.2.6</b> Explore aesthetic parameter “shape”</a></li>
<li class="chapter" data-level="5.2.7" data-path="data-visualization.html"><a href="data-visualization.html#add-axis-lables-and-plot-title-using-labs"><i class="fa fa-check"></i><b>5.2.7</b> Add axis lables and plot title using labs()</a></li>
<li class="chapter" data-level="5.2.8" data-path="data-visualization.html"><a href="data-visualization.html#change-color-pelette"><i class="fa fa-check"></i><b>5.2.8</b> Change color pelette</a></li>
<li class="chapter" data-level="5.2.9" data-path="data-visualization.html"><a href="data-visualization.html#save-the-ggplot-object-and-then-print."><i class="fa fa-check"></i><b>5.2.9</b> Save the ggplot object and then print.</a></li>
<li class="chapter" data-level="5.2.10" data-path="data-visualization.html"><a href="data-visualization.html#the-theme"><i class="fa fa-check"></i><b>5.2.10</b> The Theme</a></li>
<li class="chapter" data-level="5.2.11" data-path="data-visualization.html"><a href="data-visualization.html#adjusting-the-legend-title"><i class="fa fa-check"></i><b>5.2.11</b> Adjusting the legend title</a></li>
<li class="chapter" data-level="5.2.12" data-path="data-visualization.html"><a href="data-visualization.html#facet_wrap"><i class="fa fa-check"></i><b>5.2.12</b> Facet_wrap</a></li>
<li class="chapter" data-level="5.2.13" data-path="data-visualization.html"><a href="data-visualization.html#bar-charts"><i class="fa fa-check"></i><b>5.2.13</b> Bar charts</a></li>
<li class="chapter" data-level="5.2.14" data-path="data-visualization.html"><a href="data-visualization.html#density-plot-1"><i class="fa fa-check"></i><b>5.2.14</b> Density plot</a></li>
<li class="chapter" data-level="5.2.15" data-path="data-visualization.html"><a href="data-visualization.html#box-plot"><i class="fa fa-check"></i><b>5.2.15</b> Box plot</a></li>
<li class="chapter" data-level="5.2.16" data-path="data-visualization.html"><a href="data-visualization.html#references-1"><i class="fa fa-check"></i><b>5.2.16</b> References</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="bioconductor.html"><a href="bioconductor.html"><i class="fa fa-check"></i><b>6</b> Bioconductor</a><ul>
<li class="chapter" data-level="6.1" data-path="bioconductor.html"><a href="bioconductor.html#introduction"><i class="fa fa-check"></i><b>6.1</b> Introduction</a><ul>
<li class="chapter" data-level="6.1.1" data-path="bioconductor.html"><a href="bioconductor.html#install"><i class="fa fa-check"></i><b>6.1.1</b> Install</a></li>
<li class="chapter" data-level="6.1.2" data-path="bioconductor.html"><a href="bioconductor.html#explore-bioconductor-tutorials"><i class="fa fa-check"></i><b>6.1.2</b> Explore Bioconductor Tutorials</a></li>
<li class="chapter" data-level="6.1.3" data-path="bioconductor.html"><a href="bioconductor.html#explore-course-and-conferences-materials"><i class="fa fa-check"></i><b>6.1.3</b> Explore course and conferences materials</a></li>
<li class="chapter" data-level="6.1.4" data-path="bioconductor.html"><a href="bioconductor.html#explore-and-search-for-package-using-biocviews"><i class="fa fa-check"></i><b>6.1.4</b> Explore and Search for package using BiocViews</a></li>
<li class="chapter" data-level="6.1.5" data-path="bioconductor.html"><a href="bioconductor.html#bioconductor-forum"><i class="fa fa-check"></i><b>6.1.5</b> Bioconductor Forum</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="bioconductor.html"><a href="bioconductor.html#explore-maftools"><i class="fa fa-check"></i><b>6.2</b> Explore Maftools</a><ul>
<li class="chapter" data-level="6.2.1" data-path="bioconductor.html"><a href="bioconductor.html#about-maftools"><i class="fa fa-check"></i><b>6.2.1</b> About MAFtools</a></li>
<li class="chapter" data-level="6.2.2" data-path="bioconductor.html"><a href="bioconductor.html#about-maf-file"><i class="fa fa-check"></i><b>6.2.2</b> About MAF file</a></li>
<li class="chapter" data-level="6.2.3" data-path="bioconductor.html"><a href="bioconductor.html#install-maftools."><i class="fa fa-check"></i><b>6.2.3</b> Install maftools.</a></li>
<li class="chapter" data-level="6.2.4" data-path="bioconductor.html"><a href="bioconductor.html#load-maftools-library"><i class="fa fa-check"></i><b>6.2.4</b> Load maftools library</a></li>
<li class="chapter" data-level="6.2.5" data-path="bioconductor.html"><a href="bioconductor.html#read-example-maf-file"><i class="fa fa-check"></i><b>6.2.5</b> Read example maf file</a></li>
<li class="chapter" data-level="6.2.6" data-path="bioconductor.html"><a href="bioconductor.html#print-maf-object"><i class="fa fa-check"></i><b>6.2.6</b> Print maf object</a></li>
<li class="chapter" data-level="6.2.7" data-path="bioconductor.html"><a href="bioconductor.html#structure-of-maf-object"><i class="fa fa-check"></i><b>6.2.7</b> Structure of maf object</a></li>
<li class="chapter" data-level="6.2.8" data-path="bioconductor.html"><a href="bioconductor.html#shows-sample-summry."><i class="fa fa-check"></i><b>6.2.8</b> Shows sample summry.</a></li>
<li class="chapter" data-level="6.2.9" data-path="bioconductor.html"><a href="bioconductor.html#show-frequently-mutated-genes."><i class="fa fa-check"></i><b>6.2.9</b> Show frequently mutated genes.</a></li>
<li class="chapter" data-level="6.2.10" data-path="bioconductor.html"><a href="bioconductor.html#shows-all-fields-in-maf"><i class="fa fa-check"></i><b>6.2.10</b> Shows all fields in MAF</a></li>
<li class="chapter" data-level="6.2.11" data-path="bioconductor.html"><a href="bioconductor.html#plotting-maf-summary"><i class="fa fa-check"></i><b>6.2.11</b> Plotting MAF summary</a></li>
<li class="chapter" data-level="6.2.12" data-path="bioconductor.html"><a href="bioconductor.html#oncoplots"><i class="fa fa-check"></i><b>6.2.12</b> Oncoplots</a></li>
<li class="chapter" data-level="6.2.13" data-path="bioconductor.html"><a href="bioconductor.html#transition-and-transversions."><i class="fa fa-check"></i><b>6.2.13</b> Transition and Transversions.</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="bioconductor.html"><a href="bioconductor.html#explore-cummerbund-for-diffrential-gene-expression-analysis"><i class="fa fa-check"></i><b>6.3</b> Explore cummeRbund for Diffrential Gene Expression Analysis</a></li>
<li class="chapter" data-level="6.4" data-path="bioconductor.html"><a href="bioconductor.html#references-2"><i class="fa fa-check"></i><b>6.4</b> References</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html"><i class="fa fa-check"></i><b>7</b> R Case study and Tasks</a><ul>
<li class="chapter" data-level="7.1" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#case-study1-gene-expression-data-analysis"><i class="fa fa-check"></i><b>7.1</b> Case study1: Gene Expression Data Analysis</a><ul>
<li class="chapter" data-level="7.1.1" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#experimental-setup"><i class="fa fa-check"></i><b>7.1.1</b> Experimental setup</a></li>
<li class="chapter" data-level="7.1.2" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#objective"><i class="fa fa-check"></i><b>7.1.2</b> Objective</a></li>
<li class="chapter" data-level="7.1.3" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#steps"><i class="fa fa-check"></i><b>7.1.3</b> Steps</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#case-study1-solution"><i class="fa fa-check"></i><b>7.2</b> Case study1: Solution</a></li>
<li class="chapter" data-level="7.3" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#tasks"><i class="fa fa-check"></i><b>7.3</b> Tasks</a><ul>
<li class="chapter" data-level="7.3.1" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#vector-creation"><i class="fa fa-check"></i><b>7.3.1</b> Vector creation</a></li>
<li class="chapter" data-level="7.3.2" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#fetching-vector-elements"><i class="fa fa-check"></i><b>7.3.2</b> Fetching vector elements</a></li>
<li class="chapter" data-level="7.3.3" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#vector-manipulation"><i class="fa fa-check"></i><b>7.3.3</b> Vector manipulation</a></li>
<li class="chapter" data-level="7.3.4" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#vector-arithmetic"><i class="fa fa-check"></i><b>7.3.4</b> Vector arithmetic</a></li>
<li class="chapter" data-level="7.3.5" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#matrix-1"><i class="fa fa-check"></i><b>7.3.5</b> Matrix</a></li>
<li class="chapter" data-level="7.3.6" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#data-frame-1"><i class="fa fa-check"></i><b>7.3.6</b> Data Frame</a></li>
<li class="chapter" data-level="7.3.7" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#tasks-on-iris-data-set"><i class="fa fa-check"></i><b>7.3.7</b> Tasks on Iris data set</a></li>
<li class="chapter" data-level="7.3.8" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#visualization"><i class="fa fa-check"></i><b>7.3.8</b> Visualization</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#solutions"><i class="fa fa-check"></i><b>7.4</b> Solutions</a><ul>
<li class="chapter" data-level="7.4.1" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#vector-creation-1"><i class="fa fa-check"></i><b>7.4.1</b> Vector creation</a></li>
<li class="chapter" data-level="7.4.2" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#fetching-vector-elements-1"><i class="fa fa-check"></i><b>7.4.2</b> Fetching vector elements</a></li>
<li class="chapter" data-level="7.4.3" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#vector-manipulation-1"><i class="fa fa-check"></i><b>7.4.3</b> Vector manipulation</a></li>
<li class="chapter" data-level="7.4.4" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#vector-arithmetic-1"><i class="fa fa-check"></i><b>7.4.4</b> Vector arithmetic</a></li>
<li class="chapter" data-level="7.4.5" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#matrix-2"><i class="fa fa-check"></i><b>7.4.5</b> Matrix</a></li>
<li class="chapter" data-level="7.4.6" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#data-frame-2"><i class="fa fa-check"></i><b>7.4.6</b> Data Frame</a></li>
<li class="chapter" data-level="7.4.7" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#tasks-on-iris-data-set-1"><i class="fa fa-check"></i><b>7.4.7</b> Tasks on iris data set</a></li>
<li class="chapter" data-level="7.4.8" data-path="r-case-study-and-tasks.html"><a href="r-case-study-and-tasks.html#visualization-1"><i class="fa fa-check"></i><b>7.4.8</b> Visualization</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="descriptive-statistics.html"><a href="descriptive-statistics.html"><i class="fa fa-check"></i><b>8</b> Descriptive statistics</a><ul>
<li class="chapter" data-level="8.1" data-path="descriptive-statistics.html"><a href="descriptive-statistics.html#measure-of-centrality"><i class="fa fa-check"></i><b>8.1</b> Measure of Centrality</a></li>
<li class="chapter" data-level="8.2" data-path="descriptive-statistics.html"><a href="descriptive-statistics.html#measure-of-spread"><i class="fa fa-check"></i><b>8.2</b> Measure of Spread</a></li>
<li class="chapter" data-level="8.3" data-path="descriptive-statistics.html"><a href="descriptive-statistics.html#handle-missing-values"><i class="fa fa-check"></i><b>8.3</b> Handle missing values</a><ul>
<li class="chapter" data-level="8.3.1" data-path="descriptive-statistics.html"><a href="descriptive-statistics.html#shape-data-distribution"><i class="fa fa-check"></i><b>8.3.1</b> Shape / Data Distribution</a></li>
</ul></li>
<li class="chapter" data-level="8.4" data-path="descriptive-statistics.html"><a href="descriptive-statistics.html#estimate-skewness-and-kurtosis"><i class="fa fa-check"></i><b>8.4</b> Estimate Skewness and Kurtosis</a></li>
<li class="chapter" data-level="8.5" data-path="descriptive-statistics.html"><a href="descriptive-statistics.html#further-with-skewness-and-kurtosis."><i class="fa fa-check"></i><b>8.5</b> Further with Skewness and Kurtosis.</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="data-distributions.html"><a href="data-distributions.html"><i class="fa fa-check"></i><b>9</b> Data Distributions</a><ul>
<li class="chapter" data-level="9.1" data-path="data-distributions.html"><a href="data-distributions.html#normal-distribution"><i class="fa fa-check"></i><b>9.1</b> Normal distribution</a></li>
<li class="chapter" data-level="9.2" data-path="data-distributions.html"><a href="data-distributions.html#effect-of-mean-and-sd-parameter-on-normal-distribution"><i class="fa fa-check"></i><b>9.2</b> Effect of mean and sd parameter on normal distribution</a></li>
<li class="chapter" data-level="9.3" data-path="data-distributions.html"><a href="data-distributions.html#effect-of-n-sample-size-on-normal-distribution"><i class="fa fa-check"></i><b>9.3</b> Effect of n (sample size) on normal distribution</a></li>
<li class="chapter" data-level="9.4" data-path="data-distributions.html"><a href="data-distributions.html#use-z-score-to-calculate-percentile-area-below-or-lower-tail"><i class="fa fa-check"></i><b>9.4</b> Use z score to calculate percentile (area below or lower tail)</a></li>
<li class="chapter" data-level="9.5" data-path="data-distributions.html"><a href="data-distributions.html#use-z-score-to-calculate-upper-tail"><i class="fa fa-check"></i><b>9.5</b> Use z score to calculate Upper tail</a></li>
<li class="chapter" data-level="9.6" data-path="data-distributions.html"><a href="data-distributions.html#given-percentile-area-below-the-curve-or-lower-tail-area-find-the-z-score"><i class="fa fa-check"></i><b>9.6</b> Given percentile (area below the curve or lower tail area), find the z score</a></li>
<li class="chapter" data-level="9.7" data-path="data-distributions.html"><a href="data-distributions.html#explore-the-applet"><i class="fa fa-check"></i><b>9.7</b> Explore the applet</a></li>
<li class="chapter" data-level="9.8" data-path="data-distributions.html"><a href="data-distributions.html#evaluating-the-normal-distribution"><i class="fa fa-check"></i><b>9.8</b> Evaluating the normal distribution</a></li>
<li class="chapter" data-level="9.9" data-path="data-distributions.html"><a href="data-distributions.html#sampling-distribution"><i class="fa fa-check"></i><b>9.9</b> Sampling distribution</a></li>
<li class="chapter" data-level="9.10" data-path="data-distributions.html"><a href="data-distributions.html#standard-error"><i class="fa fa-check"></i><b>9.10</b> Standard Error</a></li>
<li class="chapter" data-level="9.11" data-path="data-distributions.html"><a href="data-distributions.html#confidence-interval"><i class="fa fa-check"></i><b>9.11</b> Confidence interval</a></li>
<li class="chapter" data-level="9.12" data-path="data-distributions.html"><a href="data-distributions.html#hypothesis-test-for-single-sample-mean"><i class="fa fa-check"></i><b>9.12</b> Hypothesis test for single sample mean</a></li>
<li class="chapter" data-level="9.13" data-path="data-distributions.html"><a href="data-distributions.html#one-tailed-test-greater"><i class="fa fa-check"></i><b>9.13</b> One tailed test (Greater)</a></li>
<li class="chapter" data-level="9.14" data-path="data-distributions.html"><a href="data-distributions.html#one-tailed-test-lesser"><i class="fa fa-check"></i><b>9.14</b> One tailed test (Lesser)</a></li>
<li class="chapter" data-level="9.15" data-path="data-distributions.html"><a href="data-distributions.html#hypothesis-test-for-two-sample-mean-unpaired"><i class="fa fa-check"></i><b>9.15</b> Hypothesis test for two sample mean, unpaired</a></li>
<li class="chapter" data-level="9.16" data-path="data-distributions.html"><a href="data-distributions.html#hypothesis-test-for-two-sample-mean-paired"><i class="fa fa-check"></i><b>9.16</b> Hypothesis test for two sample mean, paired</a></li>
<li class="chapter" data-level="9.17" data-path="data-distributions.html"><a href="data-distributions.html#check-for-normality"><i class="fa fa-check"></i><b>9.17</b> Check for normality</a></li>
<li class="chapter" data-level="9.18" data-path="data-distributions.html"><a href="data-distributions.html#check-for-uniform-variance"><i class="fa fa-check"></i><b>9.18</b> Check for uniform variance</a></li>
<li class="chapter" data-level="9.19" data-path="data-distributions.html"><a href="data-distributions.html#hypothesis-test-for-single-sample-proportion"><i class="fa fa-check"></i><b>9.19</b> Hypothesis test for single sample proportion</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="regression.html"><a href="regression.html"><i class="fa fa-check"></i><b>10</b> Regression</a><ul>
<li class="chapter" data-level="10.1" data-path="regression.html"><a href="regression.html#simple-linear-regression"><i class="fa fa-check"></i><b>10.1</b> Simple linear Regression</a><ul>
<li class="chapter" data-level="10.1.1" data-path="regression.html"><a href="regression.html#build-model"><i class="fa fa-check"></i><b>10.1.1</b> Build model</a></li>
<li class="chapter" data-level="10.1.2" data-path="regression.html"><a href="regression.html#obtain-model-summary-linear-regression-diagnostics"><i class="fa fa-check"></i><b>10.1.2</b> Obtain model summary (Linear Regression Diagnostics)</a></li>
<li class="chapter" data-level="10.1.3" data-path="regression.html"><a href="regression.html#model-coefficients"><i class="fa fa-check"></i><b>10.1.3</b> Model coefficients</a></li>
<li class="chapter" data-level="10.1.4" data-path="regression.html"><a href="regression.html#model-significance-and-coefficient-significance"><i class="fa fa-check"></i><b>10.1.4</b> Model significance and Coefficient significance</a></li>
<li class="chapter" data-level="10.1.5" data-path="regression.html"><a href="regression.html#coefficient-of-determination-or-r-squared"><i class="fa fa-check"></i><b>10.1.5</b> Coefficient of Determination or R-squared</a></li>
<li class="chapter" data-level="10.1.6" data-path="regression.html"><a href="regression.html#standard-error-and-f-statistic"><i class="fa fa-check"></i><b>10.1.6</b> Standard Error and F-Statistic</a></li>
<li class="chapter" data-level="10.1.7" data-path="regression.html"><a href="regression.html#aic-and-bic"><i class="fa fa-check"></i><b>10.1.7</b> AIC and BIC</a></li>
<li class="chapter" data-level="10.1.8" data-path="regression.html"><a href="regression.html#residuals-and-residual-plot"><i class="fa fa-check"></i><b>10.1.8</b> Residuals and Residual plot</a></li>
<li class="chapter" data-level="10.1.9" data-path="regression.html"><a href="regression.html#how-to-know-if-the-model-is-best-fit-for-your-data"><i class="fa fa-check"></i><b>10.1.9</b> How to know if the model is best fit for your data?</a></li>
<li class="chapter" data-level="10.1.10" data-path="regression.html"><a href="regression.html#lets-predict-a-new-dataset-using-the-built-model"><i class="fa fa-check"></i><b>10.1.10</b> Lets predict a new dataset using the built model</a></li>
</ul></li>
<li class="chapter" data-level="10.2" data-path="regression.html"><a href="regression.html#multiple-linear-regression"><i class="fa fa-check"></i><b>10.2</b> Multiple Linear Regression</a><ul>
<li class="chapter" data-level="10.2.1" data-path="regression.html"><a href="regression.html#build-multiple-regression-model"><i class="fa fa-check"></i><b>10.2.1</b> Build multiple regression model</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="11" data-path="clustering.html"><a href="clustering.html"><i class="fa fa-check"></i><b>11</b> Clustering</a><ul>
<li class="chapter" data-level="11.1" data-path="clustering.html"><a href="clustering.html#data-preparation"><i class="fa fa-check"></i><b>11.1</b> Data preparation</a><ul>
<li class="chapter" data-level="11.1.1" data-path="clustering.html"><a href="clustering.html#load-library"><i class="fa fa-check"></i><b>11.1.1</b> Load library</a></li>
<li class="chapter" data-level="11.1.2" data-path="clustering.html"><a href="clustering.html#load-tcga-data"><i class="fa fa-check"></i><b>11.1.2</b> Load TCGA data</a></li>
<li class="chapter" data-level="11.1.3" data-path="clustering.html"><a href="clustering.html#vizualize-data"><i class="fa fa-check"></i><b>11.1.3</b> Vizualize data</a></li>
<li class="chapter" data-level="11.1.4" data-path="clustering.html"><a href="clustering.html#data-preparation-1"><i class="fa fa-check"></i><b>11.1.4</b> Data preparation</a></li>
<li class="chapter" data-level="11.1.5" data-path="clustering.html"><a href="clustering.html#check-for-na"><i class="fa fa-check"></i><b>11.1.5</b> Check for NA</a></li>
<li class="chapter" data-level="11.1.6" data-path="clustering.html"><a href="clustering.html#scale-the-data"><i class="fa fa-check"></i><b>11.1.6</b> Scale the data</a></li>
<li class="chapter" data-level="11.1.7" data-path="clustering.html"><a href="clustering.html#vizualize-data-after-scaling"><i class="fa fa-check"></i><b>11.1.7</b> Vizualize data after scaling</a></li>
</ul></li>
<li class="chapter" data-level="11.2" data-path="clustering.html"><a href="clustering.html#clustering-1"><i class="fa fa-check"></i><b>11.2</b> Clustering</a></li>
<li class="chapter" data-level="11.3" data-path="clustering.html"><a href="clustering.html#k-means-clustering"><i class="fa fa-check"></i><b>11.3</b> K-means clustering</a><ul>
<li class="chapter" data-level="11.3.1" data-path="clustering.html"><a href="clustering.html#visualizing-k-means-clusters-in-2d-space"><i class="fa fa-check"></i><b>11.3.1</b> Visualizing k-means clusters in 2D space</a></li>
<li class="chapter" data-level="11.3.2" data-path="clustering.html"><a href="clustering.html#get-details-of-clusters"><i class="fa fa-check"></i><b>11.3.2</b> Get details of clusters</a></li>
</ul></li>
<li class="chapter" data-level="11.4" data-path="clustering.html"><a href="clustering.html#determining-the-optimal-number-of-clusters"><i class="fa fa-check"></i><b>11.4</b> Determining The Optimal Number Of Clusters</a><ul>
<li class="chapter" data-level="11.4.1" data-path="clustering.html"><a href="clustering.html#elbow-method"><i class="fa fa-check"></i><b>11.4.1</b> Elbow method</a></li>
<li class="chapter" data-level="11.4.2" data-path="clustering.html"><a href="clustering.html#average-silhouette-method"><i class="fa fa-check"></i><b>11.4.2</b> Average silhouette method</a></li>
<li class="chapter" data-level="11.4.3" data-path="clustering.html"><a href="clustering.html#gap-statistic-method"><i class="fa fa-check"></i><b>11.4.3</b> Gap statistic method</a></li>
<li class="chapter" data-level="11.4.4" data-path="clustering.html"><a href="clustering.html#nbclust-function-30-indices-for-choosing-the-best-number-of-clusters"><i class="fa fa-check"></i><b>11.4.4</b> NbClust() function: 30 indices for choosing the best number of clusters</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="12" data-path="pca.html"><a href="pca.html"><i class="fa fa-check"></i><b>12</b> PCA</a><ul>
<li class="chapter" data-level="12.1" data-path="pca.html"><a href="pca.html#about-principal-component-analysis-pca"><i class="fa fa-check"></i><b>12.1</b> About Principal Component Analysis (PCA</a></li>
<li class="chapter" data-level="12.2" data-path="pca.html"><a href="pca.html#data-preparation-2"><i class="fa fa-check"></i><b>12.2</b> Data preparation</a><ul>
<li class="chapter" data-level="12.2.1" data-path="pca.html"><a href="pca.html#load-gene-expression-data-from-tcga-breast-cancer"><i class="fa fa-check"></i><b>12.2.1</b> Load Gene expression data from TCGA Breast cancer</a></li>
<li class="chapter" data-level="12.2.2" data-path="pca.html"><a href="pca.html#data-standardization"><i class="fa fa-check"></i><b>12.2.2</b> Data standardization</a></li>
</ul></li>
<li class="chapter" data-level="12.3" data-path="pca.html"><a href="pca.html#perform-pca"><i class="fa fa-check"></i><b>12.3</b> Perform PCA</a></li>
<li class="chapter" data-level="12.4" data-path="pca.html"><a href="pca.html#visualization-and-interpretation"><i class="fa fa-check"></i><b>12.4</b> Visualization and Interpretation</a></li>
<li class="chapter" data-level="12.5" data-path="pca.html"><a href="pca.html#eigenvalues-variances-number-of-pcs-to-consider"><i class="fa fa-check"></i><b>12.5</b> Eigenvalues / Variances / Number of PCs to consider</a></li>
<li class="chapter" data-level="12.6" data-path="pca.html"><a href="pca.html#graph-of-variables"><i class="fa fa-check"></i><b>12.6</b> Graph of variables</a></li>
<li class="chapter" data-level="12.7" data-path="pca.html"><a href="pca.html#correlation-circle"><i class="fa fa-check"></i><b>12.7</b> Correlation circle</a></li>
<li class="chapter" data-level="12.8" data-path="pca.html"><a href="pca.html#quality-of-representation"><i class="fa fa-check"></i><b>12.8</b> Quality of representation</a></li>
<li class="chapter" data-level="12.9" data-path="pca.html"><a href="pca.html#referece"><i class="fa fa-check"></i><b>12.9</b> Referece</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="network-analysis.html"><a href="network-analysis.html"><i class="fa fa-check"></i><b>13</b> Network analysis</a><ul>
<li class="chapter" data-level="13.1" data-path="network-analysis.html"><a href="network-analysis.html#data-processing"><i class="fa fa-check"></i><b>13.1</b> Data processing</a><ul>
<li class="chapter" data-level="13.1.1" data-path="network-analysis.html"><a href="network-analysis.html#load-igraph-library"><i class="fa fa-check"></i><b>13.1.1</b> Load iGraph Library</a></li>
<li class="chapter" data-level="13.1.2" data-path="network-analysis.html"><a href="network-analysis.html#load-data"><i class="fa fa-check"></i><b>13.1.2</b> Load data</a></li>
</ul></li>
<li class="chapter" data-level="13.2" data-path="network-analysis.html"><a href="network-analysis.html#convert-data-frame-to-graph"><i class="fa fa-check"></i><b>13.2</b> Convert data frame to graph</a></li>
<li class="chapter" data-level="13.3" data-path="network-analysis.html"><a href="network-analysis.html#node-and-edge-details"><i class="fa fa-check"></i><b>13.3</b> Node and Edge details</a></li>
<li class="chapter" data-level="13.4" data-path="network-analysis.html"><a href="network-analysis.html#plot-parameters"><i class="fa fa-check"></i><b>13.4</b> Plot Parameters</a></li>
<li class="chapter" data-level="13.5" data-path="network-analysis.html"><a href="network-analysis.html#network-layouts"><i class="fa fa-check"></i><b>13.5</b> Network layouts</a></li>
<li class="chapter" data-level="13.6" data-path="network-analysis.html"><a href="network-analysis.html#network-and-node-descriptives"><i class="fa fa-check"></i><b>13.6</b> Network and node descriptives</a><ul>
<li class="chapter" data-level="13.6.1" data-path="network-analysis.html"><a href="network-analysis.html#edge-density"><i class="fa fa-check"></i><b>13.6.1</b> Edge density</a></li>
<li class="chapter" data-level="13.6.2" data-path="network-analysis.html"><a href="network-analysis.html#diameter"><i class="fa fa-check"></i><b>13.6.2</b> Diameter</a></li>
</ul></li>
<li class="chapter" data-level="13.7" data-path="network-analysis.html"><a href="network-analysis.html#centrality-centralization"><i class="fa fa-check"></i><b>13.7</b> Centrality &amp; centralization</a><ul>
<li class="chapter" data-level="13.7.1" data-path="network-analysis.html"><a href="network-analysis.html#degree-centrality"><i class="fa fa-check"></i><b>13.7.1</b> Degree centrality</a></li>
<li class="chapter" data-level="13.7.2" data-path="network-analysis.html"><a href="network-analysis.html#closeness"><i class="fa fa-check"></i><b>13.7.2</b> Closeness</a></li>
<li class="chapter" data-level="13.7.3" data-path="network-analysis.html"><a href="network-analysis.html#betweenness"><i class="fa fa-check"></i><b>13.7.3</b> Betweenness</a></li>
</ul></li>
<li class="chapter" data-level="13.8" data-path="network-analysis.html"><a href="network-analysis.html#hubs-and-authorities"><i class="fa fa-check"></i><b>13.8</b> Hubs and authorities</a></li>
<li class="chapter" data-level="13.9" data-path="network-analysis.html"><a href="network-analysis.html#distances-and-paths"><i class="fa fa-check"></i><b>13.9</b> Distances and paths</a></li>
<li class="chapter" data-level="13.10" data-path="network-analysis.html"><a href="network-analysis.html#cliques"><i class="fa fa-check"></i><b>13.10</b> Cliques</a></li>
<li class="chapter" data-level="13.11" data-path="network-analysis.html"><a href="network-analysis.html#community-detection"><i class="fa fa-check"></i><b>13.11</b> Community detection</a></li>
<li class="chapter" data-level="13.12" data-path="network-analysis.html"><a href="network-analysis.html#interative-network"><i class="fa fa-check"></i><b>13.12</b> Interative network</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://biosakshat.weebly.com" target="blank">Visit site</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">BioSakshat - Free Study Materials</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="bioconductor" class="section level1">
<h1><span class="header-section-number">Chapter 6</span> Bioconductor</h1>
<div id="introduction" class="section level2">
<h2><span class="header-section-number">6.1</span> Introduction</h2>
<p>The purpose of this section is to introduce you about the steps to install any R package from Bioconductor repository. We will demonstrate the installation and usage of Maftools package.</p>
<p>Bioconductor provides tools for the analysis and comprehension of high-throughput genomic data. Bioconductor uses the R statistical programming language, and is open source and open development. It has two releases each year, 1383 software packages, and an active user community.</p>
<p>Website: <a href="https://www.bioconductor.org/" class="uri">https://www.bioconductor.org/</a></p>
<div id="install" class="section level3">
<h3><span class="header-section-number">6.1.1</span> Install</h3>
<p><strong>How to install Bioconductor</strong><br />
Open R and type following commands. You must be connected to internet.<br />
source(“<a href="https://bioconductor.org/biocLite.R" class="uri">https://bioconductor.org/biocLite.R</a>”)<br />
biocLite()</p>
<p><strong>How to install bioconductor packages </strong>
biocLite(c(“GenomicFeatures”, “AnnotationDbi”))</p>
<p>Further details <a href="https://www.bioconductor.org/install/" class="uri">https://www.bioconductor.org/install/</a></p>
</div>
<div id="explore-bioconductor-tutorials" class="section level3">
<h3><span class="header-section-number">6.1.2</span> Explore Bioconductor Tutorials</h3>
<p><a href="https://www.bioconductor.org/help/workflows/" class="uri">https://www.bioconductor.org/help/workflows/</a></p>
</div>
<div id="explore-course-and-conferences-materials" class="section level3">
<h3><span class="header-section-number">6.1.3</span> Explore course and conferences materials</h3>
<p><a href="https://www.bioconductor.org/help/course-materials/" class="uri">https://www.bioconductor.org/help/course-materials/</a></p>
</div>
<div id="explore-and-search-for-package-using-biocviews" class="section level3">
<h3><span class="header-section-number">6.1.4</span> Explore and Search for package using BiocViews</h3>
<p>BiocViews allows you to browse through packages avaiable in Bioconductor. Packages are organized under different categories which will allow you to search easily.</p>
<p><a href="https://www.bioconductor.org/packages/release/BiocViews.html#___Software" class="uri">https://www.bioconductor.org/packages/release/BiocViews.html#___Software</a></p>
</div>
<div id="bioconductor-forum" class="section level3">
<h3><span class="header-section-number">6.1.5</span> Bioconductor Forum</h3>
<p><a href="https://support.bioconductor.org/" class="uri">https://support.bioconductor.org/</a></p>
<p>Here you can ask questions.</p>
</div>
</div>
<div id="explore-maftools" class="section level2">
<h2><span class="header-section-number">6.2</span> Explore Maftools</h2>
<p>For demo purpose we will explore maftools [1].</p>
<p>Suppose you are doing a large scale sequencing project where you have done whole genome sequencing for ~100 patients and you have carried out variant detection pipeline to obtain variants (SNPs, Indels) in each sample. The variant information from each patient is aggregated in MAF file (Mutation Annotation Format. Now you want to analyze this MAF file. You searched in bioconductor for any pacakge that can help you to analyze MAF file and you came across maftools.</p>
<div id="about-maftools" class="section level3">
<h3><span class="header-section-number">6.2.1</span> About MAFtools</h3>
<p>With advances in Cancer Genomics, Mutation Annotation Format (MAF) is being widley accepted and used to store somatic variants detected. The Cancer Genome Atlas Project has seqenced over 30 different cancers with sample size of each cancer type being over 200. Resulting data consisting of somatic variants are stored in the form of Mutation Annotation Format. This package attempts to summarize, analyze, annotate and visualize MAF files in an efficient manner from either TCGA sources or any in-house studies as long as the data is in MAF format [2].</p>
</div>
<div id="about-maf-file" class="section level3">
<h3><span class="header-section-number">6.2.2</span> About MAF file</h3>
<p>MAF files contain many fields ranging from chromosome names to cosmic annotations. However most of the analysis in maftools uses following fields.</p>
<p>Mandatoty fields: Hugo_Symbol, Chromosome, Start_Position, End_Position, Variant_Classification, Variant_Type and Tumor_Sample_Barcode.</p>
<p>Complete specififcation of MAF files can be found on <a href="https://wiki.nci.nih.gov/display/TCGA/Mutation+Annotation+Format+(MAF)+Specification">NCI TCGA page</a>.</p>
</div>
<div id="install-maftools." class="section level3">
<h3><span class="header-section-number">6.2.3</span> Install maftools.</h3>
<p>source(“<a href="https://bioconductor.org/biocLite.R" class="uri">https://bioconductor.org/biocLite.R</a>”)<br />
biocLite(“maftools”)</p>
</div>
<div id="load-maftools-library" class="section level3">
<h3><span class="header-section-number">6.2.4</span> Load maftools library</h3>
<div class="sourceCode" id="cb728"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb728-1" data-line-number="1"><span class="kw">library</span>(maftools);</a></code></pre></div>
</div>
<div id="read-example-maf-file" class="section level3">
<h3><span class="header-section-number">6.2.5</span> Read example maf file</h3>
<div class="sourceCode" id="cb729"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb729-1" data-line-number="1"><span class="co"># laml file path</span></a>
<a class="sourceLine" id="cb729-2" data-line-number="2">laml.maf =<span class="st"> </span><span class="kw">system.file</span>(<span class="st">&#39;extdata&#39;</span>, <span class="st">&#39;tcga_laml.maf.gz&#39;</span>, <span class="dt">package =</span> <span class="st">&#39;maftools&#39;</span>);</a>
<a class="sourceLine" id="cb729-3" data-line-number="3"><span class="co"># read laml maf file</span></a>
<a class="sourceLine" id="cb729-4" data-line-number="4">laml =<span class="st"> </span><span class="kw">read.maf</span>(<span class="dt">maf =</span> laml.maf,  <span class="dt">useAll =</span> <span class="ot">FALSE</span>);</a></code></pre></div>
<pre><code>## reading maf..</code></pre>
<pre><code>## Taking input= as a system command (&#39;zcat &lt; /home/priyabrata/R/x86_64-pc-linux-gnu-library/3.4/maftools/extdata/tcga_laml.maf.gz&#39;) and a variable has been used in the expression passed to `input=`. Please use fread(cmd=...). There is a security concern if you are creating an app, and the app could have a malicious user, and the app is not running in a secure envionment; e.g. the app is running as root. Please read item 5 in the NEWS file for v1.11.6 for more information and for the option to suppress this message.</code></pre>
<pre><code>## --Using only `Somatic` variants from Mutation_Status. Set useAll = TRUE to include everything.</code></pre>
<pre><code>## ---Oops! Mutation_Status not found. Assuming all variants are Somatic and validated.</code></pre>
<pre><code>## silent variants: 475</code></pre>
<pre><code>##         ID   N
## 1: Samples 157
## 2: 5&#39;Flank   3
## 3:     IGR   5
## 4:  Intron   8
## 5:     RNA  10
## 6:  Silent 449</code></pre>
<pre><code>## Summarizing..</code></pre>
<pre><code>##                    ID          summary  Mean Median
##  1:        NCBI_Build               37    NA     NA
##  2:            Center genome.wustl.edu    NA     NA
##  3:           Samples              192    NA     NA
##  4:            nGenes             1241    NA     NA
##  5:   Frame_Shift_Del               52 0.271      0
##  6:   Frame_Shift_Ins               91 0.474      0
##  7:      In_Frame_Del               10 0.052      0
##  8:      In_Frame_Ins               42 0.219      0
##  9: Missense_Mutation             1342 6.990      7
## 10: Nonsense_Mutation              103 0.536      0
## 11:       Splice_Site               92 0.479      0
## 12:             total             1732 9.021      9</code></pre>
<pre><code>## Gene Summary..</code></pre>
<pre><code>##       Hugo_Symbol Frame_Shift_Del Frame_Shift_Ins In_Frame_Del
##    1:        FLT3               0               0            1
##    2:      DNMT3A               4               0            0
##    3:        NPM1               0              33            0
##    4:        IDH2               0               0            0
##    5:        IDH1               0               0            0
##   ---                                                         
## 1237:      ZNF689               0               0            0
## 1238:      ZNF75D               0               0            0
## 1239:      ZNF827               1               0            0
## 1240:       ZNF99               0               0            0
## 1241:        ZPBP               0               0            0
##       In_Frame_Ins Missense_Mutation Nonsense_Mutation Splice_Site total
##    1:           33                15                 0           3    52
##    2:            0                39                 5           6    54
##    3:            0                 1                 0           0    34
##    4:            0                20                 0           0    20
##    5:            0                18                 0           0    18
##   ---                                                                   
## 1237:            0                 1                 0           0     1
## 1238:            0                 1                 0           0     1
## 1239:            0                 0                 0           0     1
## 1240:            0                 1                 0           0     1
## 1241:            0                 1                 0           0     1
##       MutatedSamples AlteredSamples
##    1:             52             52
##    2:             48             48
##    3:             33             33
##    4:             20             20
##    5:             18             18
##   ---                              
## 1237:              1              1
## 1238:              1              1
## 1239:              1              1
## 1240:              1              1
## 1241:              1              1</code></pre>
<pre><code>## Checking clinical data..</code></pre>
<pre><code>## NOTE: Missing clinical data! It is strongly recommended to provide clinical data associated with samples if available.</code></pre>
<pre><code>## Done !</code></pre>
</div>
<div id="print-maf-object" class="section level3">

<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- horizontal -->
<ins class="adsbygoogle"
     style="display:block"
     data-ad-client="ca-pub-3848514720264642"
     data-ad-slot="8864343484"
     data-ad-format="auto"
     data-full-width-responsive="true"></ins>
<script>
     (adsbygoogle = window.adsbygoogle || []).push({});
</script>


<h3><span class="header-section-number">6.2.6</span> Print maf object</h3>
<p>Summarized MAF file is stored as an MAF object. MAF object contains main maf file, summarized data and an oncomatrix which is useful to plot oncoplots (aka waterfall plots).</p>
<div class="sourceCode" id="cb743"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb743-1" data-line-number="1">laml;</a></code></pre></div>
<pre><code>## An object of class  MAF 
##                    ID          summary  Mean Median
##  1:        NCBI_Build               37    NA     NA
##  2:            Center genome.wustl.edu    NA     NA
##  3:           Samples              192    NA     NA
##  4:            nGenes             1241    NA     NA
##  5:   Frame_Shift_Del               52 0.271      0
##  6:   Frame_Shift_Ins               91 0.474      0
##  7:      In_Frame_Del               10 0.052      0
##  8:      In_Frame_Ins               42 0.219      0
##  9: Missense_Mutation             1342 6.990      7
## 10: Nonsense_Mutation              103 0.536      0
## 11:       Splice_Site               92 0.479      0
## 12:             total             1732 9.021      9</code></pre>
</div>
<div id="structure-of-maf-object" class="section level3">
<h3><span class="header-section-number">6.2.7</span> Structure of maf object</h3>
<div class="sourceCode" id="cb745"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb745-1" data-line-number="1"><span class="kw">str</span>(laml);</a></code></pre></div>
<pre><code>## Formal class &#39;MAF&#39; [package &quot;maftools&quot;] with 8 slots
##   ..@ data                          :Classes &#39;data.table&#39; and &#39;data.frame&#39;:  1732 obs. of  17 variables:
##   .. ..$ Hugo_Symbol           : chr [1:1732] &quot;ABCA10&quot; &quot;ABCA4&quot; &quot;ABCB11&quot; &quot;ABCC3&quot; ...
##   .. ..$ Entrez_Gene_Id        : int [1:1732] 10349 24 8647 8714 23 64137 64241 25 55289 92370 ...
##   .. ..$ Center                : chr [1:1732] &quot;genome.wustl.edu&quot; &quot;genome.wustl.edu&quot; &quot;genome.wustl.edu&quot; &quot;genome.wustl.edu&quot; ...
##   .. ..$ NCBI_Build            : int [1:1732] 37 37 37 37 37 37 37 37 37 37 ...
##   .. ..$ Chromosome            : int [1:1732] 17 1 2 17 6 11 2 9 2 3 ...
##   .. ..$ Start_Position        : int [1:1732] 67170917 94490594 169780250 48760974 30554429 119031351 44079555 133760430 111542370 141011154 ...
##   .. ..$ End_Position          : int [1:1732] 67170917 94490594 169780250 48760974 30554429 119031351 44079555 133760430 111542370 141011154 ...
##   .. ..$ Strand                : chr [1:1732] &quot;+&quot; &quot;+&quot; &quot;+&quot; &quot;+&quot; ...
##   .. ..$ Variant_Classification: Factor w/ 7 levels &quot;Frame_Shift_Del&quot;,..: 7 5 5 5 5 5 5 5 5 5 ...
##   .. ..$ Variant_Type          : Factor w/ 3 levels &quot;DEL&quot;,&quot;INS&quot;,&quot;SNP&quot;: 3 3 3 3 3 3 3 3 3 3 ...
##   .. ..$ Reference_Allele      : chr [1:1732] &quot;T&quot; &quot;C&quot; &quot;G&quot; &quot;C&quot; ...
##   .. ..$ Tumor_Seq_Allele1     : chr [1:1732] &quot;T&quot; &quot;C&quot; &quot;G&quot; &quot;C&quot; ...
##   .. ..$ Tumor_Seq_Allele2     : chr [1:1732] &quot;C&quot; &quot;T&quot; &quot;A&quot; &quot;T&quot; ...
##   .. ..$ Tumor_Sample_Barcode  : Factor w/ 192 levels &quot;TCGA-AB-2802&quot;,..: 171 63 190 81 111 125 96 182 140 114 ...
##   .. ..$ Protein_Change        : chr [1:1732] &quot;p.K960R&quot; &quot;p.R1517H&quot; &quot;p.A1283V&quot; &quot;p.P1271S&quot; ...
##   .. ..$ i_TumorVAF_WU         : num [1:1732] 45.7 38.1 47 56.4 41 ...
##   .. ..$ i_transcript_name     : chr [1:1732] &quot;NM_080282.3&quot; &quot;NM_000350.2&quot; &quot;NM_003742.2&quot; &quot;NM_003786.1&quot; ...
##   .. ..- attr(*, &quot;.internal.selfref&quot;)=&lt;externalptr&gt; 
##   ..@ variants.per.sample           :Classes &#39;data.table&#39; and &#39;data.frame&#39;:  192 obs. of  2 variables:
##   .. ..$ Tumor_Sample_Barcode: Factor w/ 192 levels &quot;TCGA-AB-2802&quot;,..: 190 6 146 185 114 155 45 103 118 71 ...
##   .. ..$ Variants            : int [1:192] 34 25 23 21 20 20 20 19 18 18 ...
##   .. ..- attr(*, &quot;.internal.selfref&quot;)=&lt;externalptr&gt; 
##   ..@ variant.type.summary          :Classes &#39;data.table&#39; and &#39;data.frame&#39;:  192 obs. of  5 variables:
##   .. ..$ Tumor_Sample_Barcode: Factor w/ 192 levels &quot;TCGA-AB-2802&quot;,..: 190 6 146 185 45 114 155 103 71 118 ...
##   .. ..$ DEL                 : int [1:192] 0 2 0 0 0 1 0 0 0 0 ...
##   .. ..$ INS                 : int [1:192] 6 0 0 0 1 1 1 1 1 2 ...
##   .. ..$ SNP                 : int [1:192] 28 23 23 21 19 18 19 18 17 16 ...
##   .. ..$ total               : num [1:192] 34 25 23 21 20 20 20 19 18 18 ...
##   .. ..- attr(*, &quot;.internal.selfref&quot;)=&lt;externalptr&gt; 
##   ..@ variant.classification.summary:Classes &#39;data.table&#39; and &#39;data.frame&#39;:  192 obs. of  9 variables:
##   .. ..$ Tumor_Sample_Barcode: Factor w/ 192 levels &quot;TCGA-AB-2802&quot;,..: 190 6 146 185 45 114 155 103 71 118 ...
##   .. ..$ Frame_Shift_Del     : int [1:192] 0 1 0 0 0 1 0 0 0 0 ...
##   .. ..$ Frame_Shift_Ins     : int [1:192] 5 0 0 0 1 1 1 1 0 2 ...
##   .. ..$ In_Frame_Del        : int [1:192] 0 1 0 0 0 0 0 0 0 0 ...
##   .. ..$ In_Frame_Ins        : int [1:192] 1 0 0 0 0 0 0 0 1 0 ...
##   .. ..$ Missense_Mutation   : int [1:192] 25 16 22 15 16 15 16 15 17 11 ...
##   .. ..$ Nonsense_Mutation   : int [1:192] 2 3 0 1 1 3 2 1 0 2 ...
##   .. ..$ Splice_Site         : int [1:192] 1 4 1 5 2 0 1 2 0 3 ...
##   .. ..$ total               : num [1:192] 34 25 23 21 20 20 20 19 18 18 ...
##   .. ..- attr(*, &quot;.internal.selfref&quot;)=&lt;externalptr&gt; 
##   ..@ gene.summary                  :Classes &#39;data.table&#39; and &#39;data.frame&#39;:  1241 obs. of  11 variables:
##   .. ..$ Hugo_Symbol      : chr [1:1241] &quot;FLT3&quot; &quot;DNMT3A&quot; &quot;NPM1&quot; &quot;IDH2&quot; ...
##   .. ..$ Frame_Shift_Del  : int [1:1241] 0 4 0 0 0 10 1 2 0 6 ...
##   .. ..$ Frame_Shift_Ins  : int [1:1241] 0 0 33 0 0 4 3 2 0 2 ...
##   .. ..$ In_Frame_Del     : int [1:1241] 1 0 0 0 0 0 1 0 0 2 ...
##   .. ..$ In_Frame_Ins     : int [1:1241] 33 0 0 0 0 0 0 0 0 6 ...
##   .. ..$ Missense_Mutation: int [1:1241] 15 39 1 20 18 4 8 11 15 2 ...
##   .. ..$ Nonsense_Mutation: int [1:1241] 0 5 0 0 0 8 5 1 0 1 ...
##   .. ..$ Splice_Site      : int [1:1241] 3 6 0 0 0 1 1 3 0 0 ...
##   .. ..$ total            : num [1:1241] 52 54 34 20 18 27 19 19 15 19 ...
##   .. ..$ MutatedSamples   : int [1:1241] 52 48 33 20 18 17 16 15 15 13 ...
##   .. ..$ AlteredSamples   : int [1:1241] 52 48 33 20 18 17 16 15 15 13 ...
##   .. ..- attr(*, &quot;.internal.selfref&quot;)=&lt;externalptr&gt; 
##   ..@ summary                       :Classes &#39;data.table&#39; and &#39;data.frame&#39;:  12 obs. of  4 variables:
##   .. ..$ ID     : chr [1:12] &quot;NCBI_Build&quot; &quot;Center&quot; &quot;Samples&quot; &quot;nGenes&quot; ...
##   .. ..$ summary: chr [1:12] &quot;37&quot; &quot;genome.wustl.edu&quot; &quot;192&quot; &quot;1241&quot; ...
##   .. ..$ Mean   : num [1:12] NA NA NA NA 0.271 0.474 0.052 0.219 6.99 0.536 ...
##   .. ..$ Median : num [1:12] NA NA NA NA 0 0 0 0 7 0 ...
##   .. ..- attr(*, &quot;.internal.selfref&quot;)=&lt;externalptr&gt; 
##   ..@ maf.silent                    :Classes &#39;data.table&#39; and &#39;data.frame&#39;:  475 obs. of  17 variables:
##   .. ..$ Hugo_Symbol           : chr [1:475] &quot;ABCC11&quot; &quot;ACAN&quot; &quot;ACAT1&quot; &quot;ACCN2&quot; ...
##   .. ..$ Entrez_Gene_Id        : int [1:475] 85320 176 38 41 59 284382 8728 56999 111 83440 ...
##   .. ..$ Center                : chr [1:475] &quot;genome.wustl.edu&quot; &quot;genome.wustl.edu&quot; &quot;genome.wustl.edu&quot; &quot;genome.wustl.edu&quot; ...
##   .. ..$ NCBI_Build            : int [1:475] 37 37 37 37 37 37 37 37 37 37 ...
##   .. ..$ Chromosome            : int [1:475] 16 15 11 12 10 19 5 3 3 15 ...
##   .. ..$ Start_Position        : int [1:475] 48244997 89401084 108009744 50452780 90695109 8808551 156920098 64532572 123071312 73045153 ...
##   .. ..$ End_Position          : int [1:475] 48244997 89401084 108009744 50452780 90695109 8808551 156920098 64532572 123071312 73045153 ...
##   .. ..$ Strand                : chr [1:475] &quot;+&quot; &quot;+&quot; &quot;+&quot; &quot;+&quot; ...
##   .. ..$ Variant_Classification: chr [1:475] &quot;Silent&quot; &quot;Silent&quot; &quot;Silent&quot; &quot;Silent&quot; ...
##   .. ..$ Variant_Type          : chr [1:475] &quot;SNP&quot; &quot;SNP&quot; &quot;SNP&quot; &quot;SNP&quot; ...
##   .. ..$ Reference_Allele      : chr [1:475] &quot;G&quot; &quot;C&quot; &quot;T&quot; &quot;C&quot; ...
##   .. ..$ Tumor_Seq_Allele1     : chr [1:475] &quot;G&quot; &quot;C&quot; &quot;T&quot; &quot;C&quot; ...
##   .. ..$ Tumor_Seq_Allele2     : chr [1:475] &quot;A&quot; &quot;T&quot; &quot;G&quot; &quot;G&quot; ...
##   .. ..$ Tumor_Sample_Barcode  : chr [1:475] &quot;TCGA-AB-2830&quot; &quot;TCGA-AB-2898&quot; &quot;TCGA-AB-2887&quot; &quot;TCGA-AB-3009&quot; ...
##   .. ..$ Protein_Change        : chr [1:475] &quot;p.I490I&quot; &quot;p.S1756S&quot; &quot;p.T185T&quot; &quot;p.L77L&quot; ...
##   .. ..$ i_TumorVAF_WU         : num [1:475] 34.27 38.3 49.04 48.1 0.201 ...
##   .. ..$ i_transcript_name     : chr [1:475] &quot;NM_032583.3&quot; &quot;NM_013227.2&quot; &quot;NM_000019.3&quot; &quot;NM_020039.2&quot; ...
##   .. ..- attr(*, &quot;.internal.selfref&quot;)=&lt;externalptr&gt; 
##   ..@ clinical.data                 :Classes &#39;data.table&#39; and &#39;data.frame&#39;:  192 obs. of  1 variable:
##   .. ..$ Tumor_Sample_Barcode: Factor w/ 192 levels &quot;TCGA-AB-2802&quot;,..: 190 6 146 185 114 155 45 103 118 71 ...
##   .. ..- attr(*, &quot;.internal.selfref&quot;)=&lt;externalptr&gt;</code></pre>
</div>
<div id="shows-sample-summry." class="section level3">
<h3><span class="header-section-number">6.2.8</span> Shows sample summry.</h3>
<div class="sourceCode" id="cb747"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb747-1" data-line-number="1"><span class="kw">getSampleSummary</span>(laml);</a></code></pre></div>
<pre><code>##      Tumor_Sample_Barcode Frame_Shift_Del Frame_Shift_Ins In_Frame_Del
##   1:         TCGA-AB-3009               0               5            0
##   2:         TCGA-AB-2807               1               0            1
##   3:         TCGA-AB-2959               0               0            0
##   4:         TCGA-AB-3002               0               0            0
##   5:         TCGA-AB-2849               0               1            0
##  ---                                                                  
## 188:         TCGA-AB-2933               0               0            0
## 189:         TCGA-AB-2942               0               0            0
## 190:         TCGA-AB-2946               0               0            0
## 191:         TCGA-AB-2954               0               0            0
## 192:         TCGA-AB-2982               0               0            0
##      In_Frame_Ins Missense_Mutation Nonsense_Mutation Splice_Site total
##   1:            1                25                 2           1    34
##   2:            0                16                 3           4    25
##   3:            0                22                 0           1    23
##   4:            0                15                 1           5    21
##   5:            0                16                 1           2    20
##  ---                                                                   
## 188:            0                 1                 0           0     1
## 189:            1                 0                 0           0     1
## 190:            0                 1                 0           0     1
## 191:            0                 1                 0           0     1
## 192:            0                 1                 0           0     1</code></pre>
</div>
<div id="show-frequently-mutated-genes." class="section level3">
<h3><span class="header-section-number">6.2.9</span> Show frequently mutated genes.</h3>
<div class="sourceCode" id="cb749"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb749-1" data-line-number="1"><span class="kw">getGeneSummary</span>(laml);</a></code></pre></div>
<pre><code>##       Hugo_Symbol Frame_Shift_Del Frame_Shift_Ins In_Frame_Del
##    1:        FLT3               0               0            1
##    2:      DNMT3A               4               0            0
##    3:        NPM1               0              33            0
##    4:        IDH2               0               0            0
##    5:        IDH1               0               0            0
##   ---                                                         
## 1237:      ZNF689               0               0            0
## 1238:      ZNF75D               0               0            0
## 1239:      ZNF827               1               0            0
## 1240:       ZNF99               0               0            0
## 1241:        ZPBP               0               0            0
##       In_Frame_Ins Missense_Mutation Nonsense_Mutation Splice_Site total
##    1:           33                15                 0           3    52
##    2:            0                39                 5           6    54
##    3:            0                 1                 0           0    34
##    4:            0                20                 0           0    20
##    5:            0                18                 0           0    18
##   ---                                                                   
## 1237:            0                 1                 0           0     1
## 1238:            0                 1                 0           0     1
## 1239:            0                 0                 0           0     1
## 1240:            0                 1                 0           0     1
## 1241:            0                 1                 0           0     1
##       MutatedSamples AlteredSamples
##    1:             52             52
##    2:             48             48
##    3:             33             33
##    4:             20             20
##    5:             18             18
##   ---                              
## 1237:              1              1
## 1238:              1              1
## 1239:              1              1
## 1240:              1              1
## 1241:              1              1</code></pre>
</div>
<div id="shows-all-fields-in-maf" class="section level3">
<h3><span class="header-section-number">6.2.10</span> Shows all fields in MAF</h3>
<div class="sourceCode" id="cb751"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb751-1" data-line-number="1"><span class="kw">getFields</span>(laml);</a></code></pre></div>
<pre><code>##  [1] &quot;Hugo_Symbol&quot;            &quot;Entrez_Gene_Id&quot;        
##  [3] &quot;Center&quot;                 &quot;NCBI_Build&quot;            
##  [5] &quot;Chromosome&quot;             &quot;Start_Position&quot;        
##  [7] &quot;End_Position&quot;           &quot;Strand&quot;                
##  [9] &quot;Variant_Classification&quot; &quot;Variant_Type&quot;          
## [11] &quot;Reference_Allele&quot;       &quot;Tumor_Seq_Allele1&quot;     
## [13] &quot;Tumor_Seq_Allele2&quot;      &quot;Tumor_Sample_Barcode&quot;  
## [15] &quot;Protein_Change&quot;         &quot;i_TumorVAF_WU&quot;         
## [17] &quot;i_transcript_name&quot;</code></pre>
</div>
<div id="plotting-maf-summary" class="section level3">
<h3><span class="header-section-number">6.2.11</span> Plotting MAF summary</h3>
<div class="sourceCode" id="cb753"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb753-1" data-line-number="1"><span class="kw">plotmafSummary</span>(<span class="dt">maf =</span> laml, <span class="dt">rmOutlier =</span> <span class="ot">TRUE</span>, <span class="dt">addStat =</span> <span class="st">&#39;median&#39;</span>, <span class="dt">dashboard =</span> <span class="ot">TRUE</span>);</a></code></pre></div>
<pre><code>## Warning: `legend.margin` must be specified using `margin()`. For the old
## behavior use legend.spacing</code></pre>
<pre><code>## Warning: Removed 1 rows containing non-finite values (stat_boxplot).</code></pre>
<pre><code>## Warning: `panel.margin` is deprecated. Please use `panel.spacing` property
## instead</code></pre>
<p><img src="bookdown-demo_files/figure-html/unnamed-chunk-170-1.png" width="672" /></p>
</div>
<div id="oncoplots" class="section level3">
<h3><span class="header-section-number">6.2.12</span> Oncoplots</h3>
<div class="sourceCode" id="cb757"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb757-1" data-line-number="1"><span class="co">#We will draw oncoplots for top ten mutated genes. (Removing non-mutated samples from the plot for better visualization)</span></a>
<a class="sourceLine" id="cb757-2" data-line-number="2"><span class="kw">oncoplot</span>(<span class="dt">maf =</span> laml, <span class="dt">top =</span> <span class="dv">10</span>, <span class="dt">removeNonMutated =</span> <span class="ot">TRUE</span>);</a></code></pre></div>
<p><img src="bookdown-demo_files/figure-html/unnamed-chunk-171-1.png" width="672" /></p>
</div>
<div id="transition-and-transversions." class="section level3">
<h3><span class="header-section-number">6.2.13</span> Transition and Transversions.</h3>
<div class="sourceCode" id="cb758"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb758-1" data-line-number="1">laml.titv =<span class="st"> </span><span class="kw">titv</span>(<span class="dt">maf =</span> laml, <span class="dt">plot =</span> <span class="ot">FALSE</span>, <span class="dt">useSyn =</span> <span class="ot">TRUE</span>);</a>
<a class="sourceLine" id="cb758-2" data-line-number="2"><span class="co">#plot titv summary</span></a>
<a class="sourceLine" id="cb758-3" data-line-number="3"><span class="kw">plotTiTv</span>(<span class="dt">res =</span> laml.titv);</a></code></pre></div>
<pre><code>## Warning: `legend.margin` must be specified using `margin()`. For the old
## behavior use legend.spacing</code></pre>
<p><img src="bookdown-demo_files/figure-html/unnamed-chunk-172-1.png" width="672" /></p>
</div>
</div>
<div id="explore-cummerbund-for-diffrential-gene-expression-analysis" class="section level2">
<h2><span class="header-section-number">6.3</span> Explore cummeRbund for Diffrential Gene Expression Analysis</h2>
<p>cummeRbund: Allows for persistent storage, access, exploration, and manipulation of Cufflinks high-throughput sequencing data. In addition, provides numerous plotting functions for commonly used visualizations.</p>
<p><a href="https://bioconductor.org/packages/release/bioc/html/cummeRbund.html" class="uri">https://bioconductor.org/packages/release/bioc/html/cummeRbund.html</a></p>
</div>
<div id="references-2" class="section level2">
<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- horizontal -->
<ins class="adsbygoogle"
     style="display:block"
     data-ad-client="ca-pub-3848514720264642"
     data-ad-slot="8864343484"
     data-ad-format="auto"
     data-full-width-responsive="true"></ins>
<script>
     (adsbygoogle = window.adsbygoogle || []).push({});
</script>

<h2><span class="header-section-number">6.4</span> References</h2>
<ol style="list-style-type: decimal">
<li><a href="https://bioconductor.org/packages/release/bioc/html/maftools.html" class="uri">https://bioconductor.org/packages/release/bioc/html/maftools.html</a></li>
<li>Mayakonda, A. &amp; Koeffler, H.P. Maftools: Efficient analysis, visualization and summarization of MAF files from large-scale cohort based cancer studies. bioRxiv (2016). doi: <a href="http://dx.doi.org/10.1101/052662" class="uri">http://dx.doi.org/10.1101/052662</a></li>
</ol>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="data-visualization.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="r-case-study-and-tasks.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
